### ISSUE TITLE
Feature request: a guide to styling the widget

### ISSUE THREAD

**Issue by indignant** [okta/okta-signin-widget#2283]
### I'm submitting a
  - [ ] bug report
  - [x] feature request

### Background info

Hey there 🌈 As a frontend dev, I've been tasked with styling an Okta hosted login page. The Okta documentation specifies that we can add styles in Customization tab in Okta then redirects to this repo. The README here has this text:

```
│   │   # Main CSS file for widget styles. Try not to override the classes in this
│   │   # file when creating a custom theme - the classes/elements are subject to
│   │   # change between releases
```

But I don't see where there's an example of using custom styles. I think it would be beneficial to have an example of customizing the widget without referring to the classes/elements in the markup for the widget.

### Expected behavior

As a new Okta user, I would expect some explanation or example on how to do the things the documentation says is possible in a way that is not at odds with the warnings in this repo's README.

Thank you 🦄

**Comment by shuowu** [comment]
@indignant Thanks for reporting this issue! I will reach out to our widget team for suggestions.

**Comment by shuowu** [comment]
@indignant Can you try [this guide](https://developer.okta.com/docs/guides/style-the-widget/customization-examples/). Here is also a [live widget](https://developer.okta.com/live-widget/) that you can play with.

**Comment by indignant** [comment]
Thank you for looking into this @shuowu

I should have mentioned that I had read the guide you linked to and am able to update the CSS for the page/widget if I use the IDs/classes/elements I find in the browser's inspector. However the README in this project says that this is not encouraged since "the classes/elements are subject to change between releases."

So I guess my question boils down to:
1. If I want to style the widget _and_
2. if I want to stay on the latest version of the widget ([which I think by default Okta uses the latest version](https://developer.okta.com/docs/guides/style-the-widget/style-okta-hosted/#change-the-okta-sign-in-widget-version)) _and_
3. if I want my users to experience consistent styles between releases _then_
4. how do I style the widget?

**Comment by indignant** [comment]
It seems like both the live widget editor and [this video](https://developer.okta.com/docs/guides/style-the-widget/style-self-hosted/#video-tutorial) use the classes to edit things. Is the README wrong maybe? Or is there no way to have stable styles that work between releases?

**Comment by shuowu** [comment]
@indignant For styling Okta hosted widget, you can follow [this guide](https://developer.okta.com/docs/guides/style-the-widget/style-okta-hosted/)

**Comment by indignant** [comment]
@shuowu Yep, I saw that (I actually linked to the same article in [this comment](https://github.com/okta/okta-signin-widget/issues/2283#issuecomment-957679343)). I'm having trouble explaining myself.

Resources on the Okta website suggest styling the widget using IDs/classes/elements, but the README in this repo explicitly says to avoid this:

```
│   │   # Main CSS file for widget styles. Try not to override the classes in this
│   │   # file when creating a custom theme - the classes/elements are subject to
│   │   # change between releases
```

Those things, as far as I understand them, are mutually exclusive. I can't both use the classes and avoid using the classes. So I suppose there are two options for resolving this issue:

1. If there _is not_ a stable API for styling the widget, could we get a stable API, documented it, and remove the warning in the README **OR**
2. If there _is_ a stable API, could we have it documented and remove the warning

I guess I can stay fixed to specific version of the widget, but I'd be concerned about missing important security updates. Either way, I think it would be beneficial to have a stable styling API that's documented because I'm not sure of all the possible things users might see and I want to create a consistent experience for them.

**Comment by shuowu** [comment]
@indignant Right, we will align the README file with the guide. 

Internal Ref: OKTA-443373

**Comment by jchabotamica** [comment]
@shuowu are there instructions on how to properly style the widget if you're using it as an npm module?  I'd like to remove the dependencies on the font files at a minimum!
